src/Pages/Home.js

```diff
@@ -32,8 +32,8 @@ function Home(){
                     src={require('../Assets/next.png')} 
                     alt=""/>
                 </div>
-                 {/* SocialEmail component container */}
-                 <SocialIconsAndEmail />
+                {/* SocialEmail component container */}
+                <SocialIconsAndEmail />
             </div>
             {/* About Me Section  */}
```

Comment:
Just an FYI: It’s best to reserve comments for explaining complex functionality, technical decisions, or reminders. Adding a comment above a component whose name is explicitly saying what it is/does isn’t all that helpful.
You dont need to change anything here. Just something to keep in mind for when you’re working with other team members on an app.